django/cantusdb_project/main_app/tests/test_views.py

```diff
@@ -2370,6 +2488,9 @@ def test_column_header_links(self):
             feast=feast,
             position=position,
         )
+        # incipit is automatically calculated from fulltext
+        # on chant save, so we need to set the incipit this way.
+        Chant.objects.filter(id=chant.id).update(incipit=incipit)
```

Comment:
I am confused by what's going on here. The comment is correct, but it seems to me that then your test shouldn't work around it?

Reply:
Took me a while to figure out what had been going on here - this was an adaptation of the previous method of identifying a matching search term. I've updated it to just use the beginning of the STD Spelling fulltext.